style.css

```diff
@@ -0,0 +1,3 @@
+.styles{
+    color: black;
+}
```

Comment:
You will have linters errors in your CSS file, I recommend you use these commands:

1. `npm install --save-dev hint@6.0.x`
2. `npm install --save-dev stylelint@13.3.x stylelint-scss@3.17.x stylelint-config-standard@20.0.x stylelint-csstree-validator`
3. `npx stylelint "**/*.{css,scss}"`
4. `npx stylelint "**/*.{css,scss}" --fix`

You should use these commands as you can find [HERE to set up linters for CSS and HTML](https://github.com/microverseinc/linters-config/tree/master/html-css), first to install all dependencies that linters will use, the third step is to find you linters issues in your local environment, and the fourth is to fix these issues automatically (not all) but you will spend more of your time.